.npmrc

```diff
@@ -0,0 +1,3 @@
+node-linker=hoisted
```

Comment:
Don’t [use](https://github.com/electron-userland/electron-builder/issues/683#issuecomment-241214075) [npm](http://electron.atom.io/docs/tutorial/using-native-node-modules/#using-npm) (neither .npmrc) for configuring electron headers. Use `electron-builder node-gyp-rebuild` instead.

`https://www.electron.build/configuration/configuration#configuration`

Reply:
Resolved in [5ebb156](https://github.com/baiyang-lzy/bbg/commit/5ebb156fba0a917a701e682e6b0c00aabb3193a7).